### PR TITLE
need to make this table also refresh fully not incremental

### PR DESCRIPTION
**Pull request by andrewhong5297**
related to some of the other issues with validator_stake_account_epochs_raw, this downstream table needs to also be changed from incremental to full run.

### CODE REVIEW

File: dbt_subprojects/solana/models/_sector/staking/solana/staking_solana_validator_stake_account_epochs.sql [hunk 1]
```diff
@@ -2,10 +2,8 @@
     schema = 'staking_solana'
     , alias = 'validator_stake_account_epochs'
     , materialized = 'incremental'
-    , file_format = 'delta'
-    , incremental_strategy = 'merge'
+    , file_format = 'table'
```
Comment (jeff-dude) [inline dbt_subprojects/solana/models/_sector/staking/solana/staking_solana_validator_stake_account_epochs.sql:5]:
```suggestion
    , materialized = 'table'
```

File: dbt_subprojects/solana/models/_sector/staking/solana/staking_solana_validator_stake_account_epochs.sql [hunk 2]
```diff
@@ -2,10 +2,8 @@
...
     , unique_key = ['epoch', 'epoch_time', 'stake_account', 'vote_account']
```
Comment (jeff-dude) [inline dbt_subprojects/solana/models/_sector/staking/solana/staking_solana_validator_stake_account_epochs.sql:6]:
```suggestion
```

**Review by jeff-dude — CHANGES_REQUESTED** [review]

**Comment by jeff-dude** [comment]
> related to some of the other issues with validator_stake_account_epochs_raw, this downstream table needs to also be changed from incremental to full run.

can you better help me understand why these don't work incrementally? i struggled to see when looking a bit myself

**Comment by andrewhong5297** [comment]
my best guess is it is because [in this model](https://github.com/smyyguy/spellbook/blob/18d41eb7d24ed63177c316f282e07b1c899ee7a4/dbt_subprojects/solana/models/_sector/staking/solana/staking_solana_stake_account_delegations.sql) we are getting all delegated stake accounts, but that can happen before/during an epoch. So then we are getting a bunch of stake accounts that are delegated and already added into the row, that aren't getting captured again in an incremental? 

honestly stumped, I tried looking at the full dag but didn't find anything obvious and it's slow to test because it works when you run full so I don't have a way of testing to see where the incremental went wrong (at least in the dune ui).

**Pull request merged**